boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi

```diff
@@ -132,5 +132,10 @@
 
 &adc1 {
 	pinctrl-0 = <&adc1_in13_pc4>;
+};
+
+&spi3 {
+	pinctrl-0 = <&spi3_sck_pg9 &spi3_mosi_pb5 &spi3_miso_pb4>;
+	cs-gpios = <&gpioe 0 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
```

Comment:
There was a rebase issue here as adci node lose `	status = "okay";`

Reply:
Thank you @erwango, I hope it is okay now !